CHANGELOG.rst

```diff
@@ -1,10 +1,19 @@
 Changelog
 =========
 
-0.6.0 (unreleased)
+1.0.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+**Breaking changes**
+
+- By default record attachments now contains a relative location to
```

Comment:
Do we have to have this behaviour by default? I would like to keep the old way by default if possible.

It's less work for the clients — only one request to get all the records and then just use them, versus two requests (one to get the base_url and one to get the list of records) and then doing a concatenation in the client.